engine/src/main/java/org/hibernate/search/annotations/IndexedEmbedded.java

```diff
@@ -34,9 +34,9 @@
 	String DEFAULT_NULL_TOKEN = "__DEFAULT_NULL_TOKEN__";
 
 	/**
-	 * Field name prefix, defaults to {@code propertyname.}.
+	 * Field name prefix.
 	 *
-	 * @return the field name prefix. Default to "."
+	 * @return the field name prefix. Default to ".", indicating that {@code propertyname.} should be used.
```

Comment:
I think we won't be able to agree on that. Anyway, is it okay with [the new wording](https://github.com/hibernate/hibernate-search/pull/1143/commits/cf2d367c7dd3c02ca21d558cf96f813e7426f080)?